README.md

```diff
@@ -20,11 +20,7 @@ Download the latest binaries from the [releases](https://github.com/BoltzExchang
 
 ### Docker
 
-`boltz-client` is also available as [docker image](https://hub.docker.com/r/boltz/boltz-client/tags).
-
-Example usage where your lnd macaroon and certificate are placed inside `~/.lnd`
-
-To start the build process, run:
```

Comment:
This isnt building the image, its using our published version.

Reply:
yes that's why I removed it?

Reply:
thought it was the other way because i cant remember writing it. seems like it got in there during merging

Reply:
prbly it was me, don't worry about it

Reply:
merge?